modules/cms/classes/AutoDatasource.php

```diff
@@ -100,7 +100,7 @@ public function sourceHasModel(string $source, Model $model)
 
             // Deleted paths are included as being handled by a datasource
             // The functionality built on this will need to make sure they
-            // include deleted records when actually performing sycning actions
+            // include deleted records when actually performing synching actions
```

Comment:
https://english.stackexchange.com/a/46473

Reply:
so you want that changed back? that source says that both British and Americans prefer `synching`...

Reply:
> For similar reasons and consistency, I'd recommend synced over synched and async over asynch.

Yes please

Reply:
Done @LukeTowers